packages/nx/src/project-graph/nx-deps-cache.ts

```diff
@@ -149,7 +149,10 @@ export function shouldRecomputeWholeGraph(
   if (cache.version !== '5.1') {
     return true;
   }
-  if (cache.deps['@nrwl/workspace'] !== packageJsonDeps['@nrwl/workspace']) {
+  if (
+    cache.deps['@nx/workspace'] !== packageJsonDeps['@nx/workspace'] ||
+    cache.deps['@nrwl/workspace'] !== packageJsonDeps['@nrwl/workspace']
```

Comment:
Does this catch the case where the cache.deps contains @nrwl/workspace, but the workspace just migrated to use the @nx/workspace package?

Reply:
Yup, good point.